pyface/data_view/data_models/api.py

```diff
@@ -9,3 +9,8 @@
 # Thanks for using Enthought open source!
 
 from .array_data_model import ArrayDataModel  # noqa: F401
```

Comment:
This import requires NumPy but projects using DataView may only require the RowTableDataModel. It will be annoying to have to install NumPy in order to access this `api` module for things that don't actually require NumPy.
TraitsUI api does something like this and I think it is worth doing:
https://github.com/enthought/traitsui/blob/7b53731606959dfaa7b2e3e8089d00e7a771e312/traitsui/api.py#L28-L33

(This is similar to this issue: https://github.com/enthought/pyface/issues/655)